ui/pages/swaps/build-quote/build-quote.js

```diff
@@ -627,44 +627,41 @@ export default function BuildQuote({
                   alt={t('swapSwapSwitch')}
                 />
               </Box>
-              <Typography variant={TypographyVariant.H7} marginTop={0}>
+              <Text variant={TextVariant.bodySm} as="h6" marginTop={0}>
                 {t('stxDescription')}
-              </Typography>
-              <Typography
-                as="ul"
-                variant={TypographyVariant.H7}
-                fontWeight={FONT_WEIGHT.BOLD}
-                marginTop={3}
-              >
-                <li>{t('stxBenefit1')}</li>
-                <li>{t('stxBenefit2')}</li>
-                <li>{t('stxBenefit3')}</li>
-                <li>
-                  {t('stxBenefit4')}
-                  <Typography
-                    as="span"
-                    fontWeight={FONT_WEIGHT.NORMAL}
-                    variant={TypographyVariant.H7}
-                  >
-                    {' *'}
-                  </Typography>
-                </li>
-              </Typography>
-              <Typography
-                variant={TypographyVariant.H8}
+              </Text>
+              <Text as="ul" variant={TextVariant.bodySm} marginTop={3}>
+                <strong>
+                  <li>{t('stxBenefit1')}</li>
+                  <li>{t('stxBenefit2')}</li>
+                  <li>{t('stxBenefit3')}</li>
+                  <li>
+                    {t('stxBenefit4')}
+                    <Text
+                      as="span"
+                      fontWeight={FontWeight.Normal}
+                      variant={TextVariant.bodySm}
+                    >
+                      {' *'}
+                    </Text>
+                  </li>
+                </strong>
```

Comment:
Hi there @PrgrmrHarshShukla -- i think the intent of #18651 was when there are cases where you need only a section of the text to be bold, for example:

When we put **emphasis** on one word or **multiple words** within a statement.

In this case I think the original prop of `fontWeight="bold"` makes more sense. @georgewrmarshall can correct me if I'm wrong here.

Reply:
Yes @brad-decker ,
That makes sense.

Shall I reverse the change here ?
OR shall we wait for @georgewrmarshall  to have a look ?

Reply:
Yes @brad-decker is correct that is the intended purpose of the strong tag functionality. Please revert the changes